drivers/mmcsd/mmcsd_sdio.c

```diff
@@ -2021,18 +2088,30 @@ static ssize_t mmcsd_writemultiple(FAR struct mmcsd_state_s *priv,
        */
     }
 
-  /* Send STOP_TRANSMISSION */
-
-  ret = mmcsd_stoptransmission(priv);
-  if (evret != OK)
+#ifndef CONFIG_MMCSD_MMCSUPPORT
```

Comment:
ifdef is not necessary here too if you force priv->cmd23support =1 for MMC cards.

Reply:
ditto